application/src/main/resources/thingsboard.yml

```diff
@@ -1261,6 +1261,11 @@ vc:
 
 notification_system:
   thread_pool_size: "${TB_NOTIFICATION_SYSTEM_THREAD_POOL_SIZE:10}"
+  rules:
+    trigger_types_configs:
+      NEW_PLATFORM_VERSION:
+        # In milliseconds, infinitely by default
+        deduplication_duration: "${NEW_PLATFORM_VERSION_NOTIFICATION_RULE_DEDUPLICATION_DURATION:0}"
```

Comment:
TB_NOTIFICATION_SYSTEM_NPV_DEDUPLICATION_DURATION

Reply:
Please consider refactoring this to one env variable. Something like:

notification_system.deduplication_durations: "${TB_NOTIFICATION_SYSTEM_DEDUPLICATION_DURATIONS:NEW_PLATFORM_VERSION:0}"

This will allow us to dynamically change durations without a need to update the YML file (which is not easy in the already built containers, etc. See TB_QUEUE_KAFKA_OTHER_PROPERTIES for the example and description template.